src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,136 @@
+package wolox.training.models;
+
+import com.sun.istack.NotNull;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @Column(nullable = false)
+    @NotNull
```

Comment:
I think the annotation @Column(nullable = false) is no necessary if you use @NotNull

Reply:
Yes, it was a misinterpretation of point 8 of the following blog that was recommended by the card https://www.baeldung.com/spring-dataIntegrityviolationexception

Reply:
If you use @NotNull, you don't need @Column (nullable = false)